### ISSUE TITLE
Convert PHP rules to JS

### ISSUE THREAD

**Issue by pattonwebz** [equalizedigital/accessibility-checker#875]
This is a holder to track all the conversions against. It is done when all rules are converted. Rules should be subitems.

Aria Hidden card was created before this one so linking it here: 

[Convert Aria Hidden rule to JS · Issue #873 · equalizedigital/accessibility-checker](https://github.com/equalizedigital/accessibility-checker/issues/873)

**Comment by SteveJonesDev** [comment]
PHP rules have been converted and will be release with version 1.23.0.
